apps/src/studio/bigGameInfo.js

```diff
@@ -0,0 +1,153 @@
+var Direction = require('./constants').Direction;
+
+var BigGameInfo = function (studio) {
```

Comment:
Wondering about `Info` as the suffix for what this does since there's behavior in addition to state. Hmm. Maybe `BigGameController`? `BigGameType`? `BigGameLogic`? (not sure any of those are better)